src/cdk/v2/destinations/the_trade_desk/utils.js

```diff
@@ -26,82 +33,220 @@ const getSignatureHeader = (request, secretKey) => {
   return base;
 };
 
-const responseBuilder = (items, config) => {
-  const { advertiserId, dataServer } = config;
+const prepareFromConfig = (destination) => ({
+  tracker_id: destination.Config?.trackerId,
+  adv: destination.Config?.advertiserId,
+});
+
+const getRevenue = (message) => {
+  const { event, properties } = message;
+  let revenue = properties?.value;
+  const eventsMapInfo = ECOMM_EVENT_MAP[event.toLowerCase()];
+  if (eventsMapInfo?.rootLevelPriceSupported) {
+    const { price, quantity = 1 } = properties;
```

Comment:
if quantity is not passed then we are taking the default quantity = 1. Since we are multiplying the price*quantity. 
We need some default value else revenue is considered as zero.

Reply:
Got it